tests/subsys/ipc/pbuf/prj.conf

```diff
@@ -5,3 +5,9 @@ CONFIG_ZTEST_SHUFFLE=y
 CONFIG_IPC_SERVICE=y
 CONFIG_IPC_SERVICE_ICMSG=y
 CONFIG_PBUF=y
+# pbuf is designed to run writer and reader code on two CPUs.
+# The write function cannot be preempted by the read function in a CPU,
+# or the cache management could result in data corruption.
+# In the tests, the reader and the writer are executed on a single CPU.
+# Disable data cache to prevent potential data corruption.
+CONFIG_DCACHE=n
```

Comment:
why not `CONFIG_CACHE_MANAGEMENT`?

Reply:
changed to `CONFIG_CACHE_MANAGEMENT=n`. @hubertmis @carlocaione can you take another look?